pkg/sidecarconfig/sidecarconfig.go

```diff
@@ -190,6 +191,7 @@ func TemplateSanityCheck(sidecarConfigTemplate *template.Template) error {
 				"rsyslog.k8s-integration.sfdc.com/test-volume-mounts": "test",
 				"rsyslog.k8s-integration.sfdc.com/log-volume-mounts":  "test",
 				"vault.k8s-integration.sfdc.com/vaultRole":            "test",
+				"vault.k8s-integration.sfdc.com/config":               "some:\n  - yaml\n  - array\nwith:\n  yaml: object",
```

Comment:
nit: can we define the yaml as multi line string constant and reference it in a variable called annotationValueAsYaml ?

Reply:
Done